build.sbt

```diff
@@ -69,7 +69,7 @@ val netty4LibsTest = Seq(
 val netty4Http = "io.netty" % "netty-codec-http" % netty4Version
 val netty4Http2 = "io.netty" % "netty-codec-http2" % netty4Version
 val opencensusVersion = "0.24.0"
-val jacksonVersion = "2.13.4"
+val jacksonVersion = "2.14.1"
```

Comment:
I have bad filling about such high Jackson version. Pretty sure won't work for our serving.